client/test/features/specs/support/given.queue.js

```diff
@@ -46,7 +46,7 @@ let givenQueue = (given, data)=>{
                     })
                 })
             )
-            data.apiSandbox.getStub.withArgs('/api/v1/requests/synonymbucket/ ' + data.name(data.queueIndex).replace('&',' '), sinon.match.any).returns(
+            data.apiSandbox.getStub.withArgs('/api/v1/requests/synonymbucket/+' + data.name(data.queueIndex), sinon.match.any).returns(
```

Comment:
Not my change!! It showed up after I did rebase upstream/sprint-20181122

All the changes in this commit are not mine.  Let me know if I need to fix this ...